src/Aspire.Hosting.AWS/CloudFormation/StackOutputReference.cs

```diff
@@ -0,0 +1,58 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.AWS.CloudFormation;
+
+/// <summary>
+/// A reference to an output parameter of a CloudFormation stack.
+/// </summary>
+/// <param name="name"></param>
+/// <param name="resource"></param>
+public class StackOutputReference(string name, ICloudFormationResource resource)
```

Comment:
This should implement `IValueProvider` and `IManifestExpressionProvider`

Reply:
Done